src/app/date-picker/date-picker-config.model.ts

```diff
@@ -8,6 +8,7 @@ export interface IDatePickerConfig extends IDayCalendarConfig,
                                            ITimeSelectConfig {
   closeOnSelect?: boolean;
   closeOnSelectDelay?: number;
+  apiControlOnly?: boolean;
```

Comment:
should be available in all date type components. Also, the name is a bit problematic as it currently only implies on open and close apis. So I would give it a more specific naming.